### ISSUE TITLE
Feature suggestion schedule cron weekday (Monday to Friday) 

### ISSUE THREAD

**Issue by alemens** [pschlan/cron-job.org#38]
Hi, it would be amazing if you allow to set the cron every weekday

**Comment by arxeiss** [comment]
What about user defines schedule?

![image](https://user-images.githubusercontent.com/5103109/65224807-31797780-dac4-11e9-9f2e-9d6ab00ab3fc.png)

**Comment by thorst** [comment]
I'm looking to run it every Friday at midnight. Using a user defined schedule, would I select everything in all the multi-selects, except the day of week where I would just select Friday? I feel this is a little confusing.

Maybe if you have a little description that spells out in plain English what it'll run. Or like the next 10 runs it'll do or something that would help.

What's throwing me, and probably the @alemens is that you cant deselect "days of the month."

**Comment by pschlan** [comment]
@thorst 

Every minute, our system looks at your custom schedule. If the current time is selected in *all* of the fields of your custom schedule, the job will be executed.

Example: It's Wednesday, October 9th 2019, 08:44.
1. Check if 9 is selected in "Day of month"
2. Check if Wednesday is selected in "Days of week"
3. Check if 10 (= October) is selected in "Months"
4. Check if 8 is selected in "Hours"
5. Check if 44 is selected in "Minutes"

If all checks are positive, the job is executed.

Let's assume you create your schedule like you described, but you de-select all "Days of month".
It's Friday, October 11th, 00:00.
1. Check if 11 is selected in "Day of month" -> :x:
2. Check if Firday is selected in "Days of week" -> :white_check_mark:
3. Check if 10 (= October) is selected in "Months" -> :white_check_mark:
4. Check if 0 is selected in "Hours" -> :white_check_mark:
5. Check if 0 is selected in "Minutes" -> :white_check_mark:
--> Since not all checks are positive, the job is NOT executed. This is why all days of month have to be selected: Because you don't want to exclude any day of month from your schedule. (If you want to execute it every Friday except if the Friday is, for example, the 4th day of the month, you would de-select 4 in "Days of month".)
